### PR TITLE
Rename project -> app throughout docs

### PR DESCRIPTION
**Pull request by archiewood**
### Description

We now refer to Evidence sites as apps, not projects. 
This updates the docs site for this change. 

There are places we also need to make this change:
- [x] Marketing site (httos://evidence.dev) https://github.com/evidence-dev/marketing-site/pull/267
- [ ] ~~VS Code extension (Commands)~~ https://github.com/evidence-dev/evidence/pull/2633
- [ ] ~~Template repo (package.json, instructions, readme)~~ https://github.com/evidence-dev/template/pull/173
- [ ] ~~Evidence Cloud UI~~

**Note: post agreement that projects refer to the "collection of code files on your computer", the PRs for the extension and template are not needed. The Cloud UI can be addressed separately.**

### Checklist

- [ ] For UI or styling changes, I have added a screenshot or gif showing before & after
- [ ] I have added a [changeset](https://github.com/evidence-dev/evidence/blob/main/CONTRIBUTING.md#adding-a-changeset)
- [x] I have added to the docs where applicable
- [ ] I have added to the [VS Code extension](https://github.com/evidence-dev/evidence-vscode) where applicable

### CODE REVIEW

File: sites/docs/pages/components/custom-components.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ We'd love to see what you've built, and may add generally applicable components
 
 If you were creating a component called `<Hello />`, which included some text and a BarChart, to use in `index.md`, you could do so like this:
 
-Add a folder called `components/` in the root of your project. This is where Evidence looks for your Svelte components:
+Add a folder called `components/` in the root of your app directory. This is where Evidence looks for your Svelte components:
```
Comment (archiewood) [inline sites/docs/pages/components/custom-components.md:30]: This is the one scenario where I am unsure about the change. When you are referring to the directory of files. Is this still an app, or is this a _project_ that you use to build your app...

File: sites/docs/pages/core-concepts/data-sources/index.md [hunk 2]
```diff
@@ -378,15 +378,15 @@ Where `[your_tab_name]` is the name of the tab in your Google Sheet, with spaces
...
+Get started by selecting the `CSV` connector on the Settings page in your app, naming it and then clicking "confirm changes". 
 
 Then copy any CSV files you want to query into `sources/[your_csv_source_name]/`. Your source names and csv files can only contain letters, numbers and underscores eg `/my_source/my_csv_2024.csv`
 
 The section below applies to both CSV and Parquet files.
 
 #### How to Query a CSV File
 
-Evidence looks for CSV files stored in a `sources/[your_csv_source_name]/` folder in the root of your Evidence project. You can query them using this syntax:
+Evidence looks for CSV files stored in a `sources/[your_csv_source_name]/` folder in the root of your Evidence app. You can query them using this syntax:
```
Comment (archiewood) [inline sites/docs/pages/core-concepts/data-sources/index.md:389]: same question

File: sites/docs/pages/components/custom-components.md (hunk 1 above)
Comment (archiewood) [reply sites/docs/pages/components/custom-components.md:30]: perhaps `app workspace`

**Comment by mcrascal** [comment]
Renaming the built site to "app" makes a lot of sense to me. 

The place where this feels a bit ambiguous is when referring to files "inside" the app. "Edit index.md in the app". 

It'd feel more natural to me to refer to the collection of files on your machine as a project. The index.md file is in the project, the home page is in the app.

**Comment by archiewood** [comment]
@mcrascal how would you feel about this "collection of files in a folder" being called the "workspace"

similar to [VSCode's definition](https://code.visualstudio.com/docs/editor/workspaces)

Open `pages/index.md` in your workspace.

**Comment by mcrascal** [comment]
> how would you feel about this "collection of files in a folder" being called the "workspace"

@archiewood it'd feel like a contortion to say "open the index.md file in your workspace", and not "open the index.md file in your project".

What is _wrong_ with project?

**Comment by archiewood** [comment]
I'm not against it, but felt like workspace was perhaps clearer.

Evidence Cloud has the concept of projects too, but perhaps those become apps so no issue there.

Happy to stick with project. Will pick through these PRs to delineate.

File: sites/docs/pages/core-concepts/data-sources/index.md (hunk 2 above)
Comment (archiewood) [inline sites/docs/pages/core-concepts/data-sources/index.md:389]:
```suggestion
Evidence looks for CSV files stored in a `sources/[your_csv_source_name]/` folder in the root of your Evidence project. You can query them using this syntax:
```

File: sites/docs/pages/components/base-map.md [hunk 3]
```diff
@@ -306,9 +306,9 @@ required
 options="URL"
 >
 
-Path to source geoJSON data from - can be a URL (see [Map Resources](#map-resources)) or a file in your project. 
+Path to source geoJSON data from - can be a URL (see [Map Resources](#map-resources)) or a file in your app's `static` directory.
```
Comment (archiewood) [inline sites/docs/pages/components/base-map.md:309]:
```suggestion
Path to source geoJSON data from - can be a URL (see [Map Resources](#map-resources)) or a file in your project.
```

File: sites/docs/pages/components/custom-components.md [hunk 4]
```diff
@@ -91,12 +91,12 @@ In the custom component:
...
 
 ## Optional: Publishing Your Components as a Plugin
-If you have built custom components that you would like other Evidence users to be able to use in their projects, you can publish them as an Evidence plugin. See the [Plugin section](/plugins/create-component-plugin/) for more details.
+If you have built custom components that you would like other Evidence users to be able to use in their apps, you can publish them as an Evidence plugin. See the [Plugin section](/plugins/create-component-plugin/) for more details.
 
 ## Utility Functions
 Evidence provides a collection of helpful utility functions to use within custom components, for things like error handling, data manipulation, and value formatting.
 
-To use these in your project, you must import them explicitly in the script tag portion of your component. The import line for each utlity function is included for reference below:
+To use these in your app, you must import them explicitly in the script tag portion of your component. The import line for each utlity function is included for reference below:
```
Comment (archiewood) [inline sites/docs/pages/components/custom-components.md:99]:
```suggestion
To use these utilities, you must import them explicitly in the script tag portion of your component. The import line for each utlity function is included for reference below:
```

File: sites/docs/pages/core-concepts/formatting/index.md [hunk 5]
```diff
@@ -41,7 +41,7 @@ Formatting does not apply to the date axis of a chart. For example, if you set `
 #### Reusable Formats
 For a more reusable approach, you can use [SQL format tags](#sql-format-tags), which let you define formats within your SQL. This guarantees that your columns will be formatted in the same way wherever they are used in Evidence.
 
-You can also create your own [custom formats](#custom-formats), which are format codes you can reuse across your project.
+You can also create your own [custom formats](#custom-formats), which are format codes you can reuse across your app.
```
Comment (archiewood) [inline sites/docs/pages/core-concepts/formatting/index.md:44]:
```suggestion
You can also create your own [custom formats](#custom-formats), which are format codes you can reuse across your project.
```

File: sites/docs/pages/core-concepts/queries/index.md [hunk 6]
```diff
@@ -95,7 +95,7 @@ To use sql file queries, you need to place them in the `queries` directory, and
 An example setup could be:
 
 ```
-my-evidence-project/
+my-evidence-app/
```
Comment (archiewood) [inline sites/docs/pages/core-concepts/queries/index.md:98]:
```suggestion
my-evidence-project/
```

**Comment by archiewood** [comment]
I've processed the changes we discussed. @ItsMeBrianD you may wish to check you are happy with the roadmap

**Review by ItsMeBrianD — APPROVED** [review]

**Pull request merged**